### PR TITLE
Updated vylogs from mini pager to full pager.

### PR DESCRIPTION
**Pull request by bharath-kondeti**
**Related Issue/Ticket:**

https://github.com/ymcatwincities/openy_gated_content/issues/72

## Steps to test:

- [ ] Take a git pull
- [ ] Then run drush updb -y
- [ ] Enable "Open Y Virtual YMCA Log" module.
- [ ] Navigate to Logs from menu. Menu Path: Manage >> Virtual Y >> Logs.
- [ ] Then check the pager will be updated to full.

### CODE REVIEW

File: modules/openy_gc_log/config/install/views.view.virtual_y_logs.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+uuid: 9e78fd87-e8b0-44e6-9621-88c3b895b38f
```
Comment (froboy) [inline modules/openy_gc_log/config/install/views.view.virtual_y_logs.yml:1]: UUID should not be in exported config.

File: modules/openy_gc_log/config/install/views.view.virtual_y_logs.yml [hunk 2]
```diff
@@ -646,13 +650,9 @@ display:
                 operator: contains
                 value: entityView
               3:
-                title: videoPlaybackStarted
```
Comment (froboy) [inline modules/openy_gc_log/config/install/views.view.virtual_y_logs.yml:649]: It looks like these lines might have been unintentionally removed.

File: modules/openy_gc_log/openy_gc_log.install [hunk 3]
```diff
@@ -91,3 +91,17 @@ function openy_gc_log_update_8004() {
     'views.view.virtual_y_logs',
   ]);
 }
+
+/**
+ * Update logs view.
```
Comment (froboy) [inline modules/openy_gc_log/openy_gc_log.install:96]: Great!

**Review by froboy — CHANGES_REQUESTED** [review]
@bharath-kondeti I've reviewed and the functionality looks good, but I've noted a few requested changes in the code.
Virtual_Y_Logs___Open_Y_Sandbox_-_carnation

Also, please update your testing instructions based on testing from the builds that are generated above. I had to first enable the log module then log in and hit a number of pages as a member in order to test the new functionality.

Thanks!

**Review by froboy — APPROVED** [review]
Looks good to me. Thanks!

**Comment by anpolimus** [comment]
@bharath-kondeti thank you for upgrade path!
Approved, sending to QA.

**Comment by NadyaRudenko** [comment]
retest

**Comment by NadyaRudenko** [comment]
Tested, all good, thanks

**Pull request merged**